lib/celluloid/eventsource.rb

```diff
@@ -31,7 +41,7 @@ def initialize(uri, options = {})
       @last_event_id = String.new
 
       @reconnect_timeout = 1
-      @on = { open: ->{}, message: ->(_) {}, error: ->(_) {} }
+      @on = { open: ->{}, message: ->(_) {}}
```

Comment:
I removed this error handler receiver or whatever it's called since the only errors we were sending were related to the connection, which we now just indefinitely retry.